.byebug_history

```diff
@@ -1,107 +0,0 @@
-continue
```

Comment:
You can ignore my previous comment...just saw this commit. If you want, you might want to interactive rebase your commits and make this comment be part of the commit where this was accidentally checked in. This way the Git history will be cleaned up to make it look like this was never checked in. Example (i.e. `git rebase --interactive "@~4`):

```
pick 96773f3 replaces byebug with pry-byebug
fixup 2579acd removes byebug history

# Rebase a3a10ab..2579acd onto a3a10ab (4 commands)
#
# Commands:
# p, pick = use commit
# r, reword = use commit, but edit the commit message
# e, edit = use commit, but stop for amending
# s, squash = use commit, but meld into previous commit
# f, fixup = like "squash", but discard this commit's log message
# x, exec = run command (the rest of the line) using shell
# d, drop = remove commit
```